Description: Fix no more working "sort +<n>" syntax in arp2ethers
 (Yes, the "2" needs to be replaced with at "3". See
 https://en.wikipedia.org/wiki/Sort_(Unix)#Columns_or_fields)
Author: Axel Beckert <abe@debian.org>
Bug-Debian: https://bugs.debian.org/545669

Index: arpwatch/arp2ethers
===================================================================
--- arpwatch.orig/arp2ethers	2015-06-13 22:59:32.464242594 +0200
+++ arpwatch/arp2ethers	2015-06-13 23:00:59.832637868 +0200
@@ -28,7 +28,7 @@
 	exit 1
 }
 
-sort +2rn $FILE | \
+sort -rnk3 $FILE | \
     awk 'NF == 4 { print }' | \
 # 1999-04-12	KELEMEN Peter <fuji@debian.org>
 #    awk -f p.awk | \
